File: src/transformers/features/http/CreateHttpValidateFormDataTransformer.ts
import { HttpValidateFormDataProgrammer } from "../../../programmers/http/HttpValidateFormDataProgrammer";

import { GenericTransformer } from "../../internal/GenericTransformer";

export namespace CreateHttpValidateFormDataTransformer {
  export const transform = GenericTransformer.factory(
    "http.createValidateFormData",
  )(
    (project) => (modulo) =>
      HttpValidateFormDataProgrammer.write(project)(modulo),
  );
}

File: packages/transform/src/features/http/CreateHttpValidateFormDataTransformer.go
package http

import (
	corehttp "github.com/samchon/typia/packages/core/src/programmers/http"
	internal "github.com/samchon/typia/packages/transform/src/internal"
)

var CreateHttpValidateFormDataTransformer = createHttpValidateFormDataTransformerNamespace{}

type createHttpValidateFormDataTransformerNamespace struct{}

func (createHttpValidateFormDataTransformerNamespace) Transform(call internal.ITypiaCallExpression) string {
	return corehttp.HttpValidateFormDataProgrammer.Write(call.TypeText)
}
